pkg/odo/cli/component/delete.go

```diff
@@ -180,6 +181,11 @@ func (do *DeleteOptions) Run() (err error) {
 		}
 	}
 
+	// If request is to delete non existing component exit with exit code 1
+	if !do.isCmpExists && !do.componentDeleteAllFlag {
+		os.Exit(1)
```

Comment:
should we just exit? shouldn't we show an error here?

Reply:
Thanks for pointing, Updated